db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :ip_or_hostname
+      t.string :name
+      t.text :labels
+      t.belongs_to :container_deployment, :type => :bigint
+      t.belongs_to :vm,                   :type => :bigint
+      t.text :docker_storage_devs, :array => true, :default => []
+      t.string :docker_storage_data_size
```

Comment:
I'm not sure we want to specify data size here, rather the defaults should probably be used instead.

There should probably be a field here to docker_storage_vg even if it isn't exposed for configuration and defaults to some specific value. If that value isn't set in docker-storage-setup, then it will attempt to use the root VG, which wouldn't be available for cloud installations.